packages/auth/src/authClients/magicLink.ts

```diff
@@ -18,6 +18,8 @@ export const magicLink = (client: MagicLink): AuthClientMagicLink => {
     logout: async () => {
       await client.user.logout()
     },
+    signup: async ({ email, showUI }) =>
+      await client.auth.loginWithMagicLink({ email: email, showUI: showUI }),
```

Comment:
```suggestion
      await client.auth.loginWithMagicLink({ email, showUI }),
```

Reply:
The `loginWithMagicLink` is the same as `login` so I will change both `login` and `signup` here.